### DISCUSSION TITLE
Push GitHub Container Registry

### DISCUSSION THREAD

**Discussion by vitorafgomes** [dotnet/sdk-container-builds#500] [Q&A]
Hi, could anyone help me?
For me to publish to ghcr.io, do I need anything else via git action?

      - name: Dotnet Push GitHub Container Registry
        run: dotnet publish --os linux --arch arm64 /t:PublishContainer -p:ContainerImageTag=${{ steps.previoustag.outputs.tag }} -c Release
        env:
          SDK_CONTAINER_DEBUG_REGISTRY_FORCE_CHUNKED_UPLOAD: 'true'
          SDK_CONTAINER_REGISTRY_UNAME: ${{ github.repository_owner }}
          SDK_CONTAINER_REGISTRY_PWORD: ${{ secrets.TOKEN_GITHUB }}
          SDK_CONTAINER_REGISTRY_SERVER: ghcr.io
          
 
![image](https://github.com/dotnet/sdk-container-builds/assets/5174765/92366e81-2a35-44e7-b8ca-c4a78713a973)

**Comment by baronfel** [comment]
The most important thing is to specify the ContainerRegistry property - this tells the tooling where you'd like to push the generated image.

**Reply by baronfel** [reply]
You can see an example of this here: https://github.com/dotnet/sdk-container-builds/blob/main/docs/ContainerCustomization.md

**Comment by vitorafgomes** [comment]
I added the properties in .csproj

![image](https://github.com/dotnet/sdk-container-builds/assets/5174765/bebdf9e4-4a69-43a9-b2a2-83a50e786dce)

And in GitAction I just Login in to Docker

![image](https://github.com/dotnet/sdk-container-builds/assets/5174765/593adc70-fdfb-497a-9f85-ae2a3c4537eb)
